README.md

```diff
@@ -157,6 +157,34 @@ export default class MoreFriendz extends Component {
 }
 ```
 
+```js
+// wrapping ember-concurrency tasks
+import Component from '@ember/component';
+import { task } from 'ember-concurrency';
+import { waitFor } from 'ember-test-waiters';
+
+export default Component.extend({
+  doTaskStuff: task(waitFor(function* doTaskStuff() {
+    yield somethingAsync();
+  }
+});
+```
+
+```js
+// decorator form with ember-concurrency tasks
+import Component from '@ember/component';
+import { task } from 'ember-concurrency-decorators';
+import { waitFor } from 'ember-test-waiters';
+
+export default class Friendz extends Component {
+  @task
+  @waitFor
```

Comment:
Can we add a comment that helps explain the relevance of ordering WRT these decorators?